lib/types/local-date.js

```diff
@@ -40,61 +50,106 @@ class LocalDate {
             typeof month === "number" &&
             typeof day === "number"
         ) {
-            // Use setUTCFullYear as if there is a 2 digit year, Date.UTC() assumes
-            // that is the 20th century.
-            this.date = new Date();
-            this.date.setUTCHours(0, 0, 0, 0);
-            this.date.setUTCFullYear(year, month - 1, day);
-            if (isNaN(this.date.getTime())) {
+            // it will throw an error if the values are wrong. TODO: add tests for this.
```

Comment:
1. Sentences start with uppercase
2. TODO - add those tests or remove this TODO